plugin/src/main/java/net/dzikoysk/funnyguilds/config/PluginConfiguration.java

```diff
@@ -1144,6 +1144,14 @@ public static class LivesRepeatingSymbol extends OkaeriConfig {
     @Comment("Jak długo plugin powinien czekać na zatrzymanie wszystkich bieżących zadań przy wyłączaniu serwera")
     public Duration pluginTaskTerminationTimeout = Duration.ofSeconds(30);
 
+    @Comment("")
+    @Comment("Jaki argument powinien zostać podany przez gracza gdy chce zaprosić wszystkich graczy w danym promieniu")
+    public String inviteCommandAllArgument = "*";
+
+    @Comment("")
+    @Comment("Jaki jest maksymalny promień podczas zapraszania graczy w danym promieniu")
+    public Double inviteCommandAllMaxRange = Math.pow(10.0, 2);
```

Comment:
Dlaczego to jest Double, a nie double? Oraz dodaj do tego fielda _ na koniec (dodaj tylko adnotacje @CustomKey("invite-command-all-max-range") i field o tej samej nazwie co ten aktualnie tylko dodaj do niego adnotacje @Exclude). I podnoś do potęgi w load() ponieważ aktualnie to podniesienie do potęgi i zapisze do configu i za każdym razem będzie podnosić do potęgi.

Reply:
Zamiast robić te okropne z hacki z dublowaniem fieldów, _silnie zalecam_ zrezygnowanie z jakiegokolwiek używania potęg przy kodzie od tej funkcji, wyjaśnienie w moim review — zbędna mikrooptymalizacja.